packages/buendia-backup/data/usr/bin/buendia-backup

```diff
@@ -35,6 +35,12 @@ if [ "$root" = "" -o "$1" = "-h" ]; then
     exit 1
 fi
 
+# Check if there's anything to back up. If not, notify and exit early.
+if buendia-db-is-unused >/dev/null; then
+    echo "No patients have been created; skipping backup."
```

Comment:
We'll probably make adjustments to how the db-is-unused script works, so don't bake its implementation ("No patients") into the message -- just say "Database is pristine" or "empty" or similar. (See other comments about "unused".)

Note also that this may extend beyond the database -- backups include the profile directory as well, so it might be worth backing those up even if no patients have been added yet.  But perhaps for down the road.